src/Resources/contao/widgets/ListWizard.php

```diff
@@ -115,117 +118,21 @@ public function generate()
 	 * @param DataContainer $dc
 	 *
 	 * @return string
+	 *
+	 * @throws \Exception
+	 * @throws \Contao\CoreBundle\Exception\RedirectResponseException
+	 *
+	 * @deprecated Deprecated since Contao 4.3 to be removed in 5.0. Use the BackendCsvImportController instead.
```

Comment:
Shouldn’t this be “Use the contao.controller.backend_csv_import service instead”?